### PR TITLE
Cheatsheets for Dockerfile added.

### PR DESCRIPTION
**Pull request by atilacamurca**
**What does your Instant Answer do?**
Show instructions used in a Dockerfile file.

**What is the data source for your Instant Answer? (Provide a link if possible)**
<http://docs.docker.com/engine/reference/builder/>

**Why did you choose this data source?**
It's in the main page of the docker project.

**Are there any other alternative (better) data sources?**
not that I know

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
developers

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
<https://duck.co/ideas/idea/6700/dockerfile-cheatsheet>

**Are you having any problems? Do you need our help with anything?**
no, but I really appreciate if you guys have a docker image.

**Where did you hear about DuckDuckHack? (For first time contributors)**
In the website <https://duckduckgo.com/about>

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
dockerfile-cheatsheet

-----
IA Page: https://duck.co/ia/view/dockerfile_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@atilacamurca Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

File: share/goodie/cheat_sheets/json/dockerfile.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
+{
+    "id": "dockerfile_cheat_sheet",
+    "name": "Dockerfile",
+    "description": "Text document that contains all the commands a user could call on the command line to assemble an image",
+    "metadata": {
+        "sourceName": "Dockerfile reference",
+        "sourceUrl": "http://docs.docker.com/engine/reference/builder/"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/dockerfile.json]: It looks like they support HTTPS, let's send the user there

File: share/goodie/cheat_sheets/json/dockerfile.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
+{
+    "id": "dockerfile_cheat_sheet",
+    "name": "Dockerfile",
+    "description": "Text document that contains all the commands a user could call on the command line to assemble an image",
+    "metadata": {
+        "sourceName": "Dockerfile reference",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/dockerfile.json]: Usually this references the domain name. I think we can change this to `Docker` or `Docker Docs`?

**Comment by moollaza** [comment]
@atilacamurca Thanks a lot for contributing, we really appreciate it! I've left you some feedback that needs to be implemented. Once things are updated please let us know by leaving a comment, so we can review this again. If you have any questions, feel free to ask them here!

**Comment by moollaza** [comment]
Also, this has now been deployed to beta: https://beta.duckduckgo.com/?q=dockerfile+cheat+sheet&ia=cheatsheet&iax=1

**Comment by atilacamurca** [comment]
hello @moollaza, I just committed the needed changes.

**Comment by moollaza** [comment]
@atilacamurca this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@atilacamurca congrats! It's live: https://duckduckgo.com/?q=Dockerfile+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**